code-challenges.js

```diff
@@ -7,9 +7,25 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
 
-
-
-
+// I notice that there is never "purple" so I make sure to initiate my function using .shift() first.
+// call a function that takes an array of strings and deletes the first color "purple"
+//
+
+const stringDelShuf = (string) => {
+    string.shift()
+  return string.sort((a,b) => 0.5 - Math.random())
+  let stringDelShuf2 = (string) => {
```

Comment:
these lines of code will never run because there is a return above. once that return runs, the function stops